doc/nrf/app_dev/bootloaders_dfu/qspi_xip_split_image.rst

```diff
@@ -217,8 +217,7 @@ Programming with the QSPI XIP split image
 Programming of the application is supported using the :ref:`standard procedure <programming>`.
 The standard procedure will program the firmware using the default nrfjprog configuration which, for QSPI, is PP4IO mode.
 
-.. note::
-      |nrfjprog_deprecation_note|
+##need edits
```

Comment:
@nordicjm @carlescufi Hi, could you provide inputs on how to do the same without using nrfjprog? We are deprecating this in ncs 3.0